app/services/metrics/number_of_pdfs_metric.rb

```diff
@@ -0,0 +1,38 @@
+module Metrics
+  class NumberOfPdfsMetric
+    PDF_XPATH = "//*[contains(@class, 'attachment-details')]//a[contains(@href, '.pdf')]".freeze
+
+    def initialize(content_item)
+      @content_item = content_item
+    end
+
+    def run
+      documents_string = extract_documents
+      documents = parse documents_string
+      { number_of_pdfs: number_of_pdfs(documents) }
+    end
+
+  private
+
+    def extract_documents
+      details = @content_item[:details].symbolize_keys
+      documents = []
+      document_keys.each do |k|
+        documents.push(details.fetch(k)) if details.key?(k)
+      end
+      documents.join('')
+    end
+
+    def document_keys
+      %i(documents final_outcome_documents)
```

Comment:
You're right, we would get a more accurate representation of unique pdfs attached to a content item, but i don't think that's something we care about; this is about pdf heavy content, so whether unique or not it still should still take them into account.